core/controllers/payload_validator.py

```diff
@@ -110,3 +110,128 @@ def convert_string_to_bool(param: str) -> Optional[Union[bool, str]]:
+    'NewSkillHandler',
+    'NewTopicHandler',
+    'NotificationHandler',
+    'NotificationsDashboardHandler',
+    'NotificationsDashboardPage',
+    'NotificationsHandler',
+    'OldNotificationsDashboardRedirectPage',
+    'PendingAccountDeletionPage',
+    'PreferenceHandler',
+    'PreferencesHandler',
+    'ProfileHandler',
+    'ProfilePage',
+    'QuebstionsListHandler',
+    'QuestionCountDataHandler',
+    'QuestionCreationHandler',
+    'QuestionPlayerHandler',
+    'QuestionSkillLinkHandler',
+    'QuestionsListHandler',
+    'ReaderFeedbackHandler',
+    'RecentCommitsHandler',
+    'RecommendationsHandler',
+    'ResubmitSuggestionHandler',
+    'ReviewableSuggestionsHandler',
+    'SignupHandler',
+    'SignupPage',
+    'SiteLanguageHandler',
+    'SkillDataHandler',
+    'SkillDescriptionHandler',
+    'SkillMasteryDataHandler',
+    'SkillsDashboardPageDataHandler',
+    'SolutionHitEventHandler',
+    'StartedTranslationTutorialEventHandler',
+    'StateCompleteEventHandler',
+    'StateHitEventHandler',
+    'StoryUrlFragmentHandler',
+    'SubtopicPageDataHandler',
+    'SubtopicViewerPage',
+    'SuggestionEmailHandler',
+    'SuggestionHandler',
+    'SuggestionListHandler',
+    'SuggestionToExplorationActionHandler',
+    'SuggestionToSkillActionHandler',
+    'SuggestionsProviderHandler',
+    'TopicAssignmentsHandler',
+    'TopicEditorPage',
+    'TopicEditorStoryHandler',
+    'TopicNameHandler',
+    'TopicPageDataHandler',
+    'TopicPublishHandler',
+    'TopicPublishSendMailHandler',
+    'TopicRightsHandler',
+    'TopicUrlFragmentHandler',
+    'TopicViewerPage',
+    'TopicsAndSkillsDashboardPage',
+    'TopicsAndSkillsDashboardPageDataHandler',
+    'UnsentFeedbackEmailHandler',
+    'UpdateQuestionSuggestionHandler',
+    'UpdateTranslationSuggestionHandler',
+    'UrlHandler',
+    'UserInfoHandler',
+    'UserSubmittedSuggestionsHandler',
+    'UsernameCheckHandler',
+    'ValidateExplorationsHandler',
+    'ValueGeneratorHandler',
+    'VoiceArtistManagementHandler',
+    'OppiaMLVMHandler',
+    # Oppia Root page is the unified entry for page routes to the frontend.
+    # So, it should exempted from schema validation.
+    'OppiaRootPage',
+    'CsrfTokenHandler',
+    'Error404Handler',
+    'FrontendErrorHandler',
+    'WarmupPage',
+    'HomePageRedirectPage',
+    'SplashRedirectPage'
+    ]
+
+# These handlers do not require any schema validation.
+HANDLER_CLASS_NAMES_WHICH_DO_NOT_REQUIRE_SCHEMAS = [
+    'SessionBeginHandler',
+    'SessionEndHandler',
+    'SeedFirebaseHandler'
+]
+
+# HANDLER_CLASS_NAMES_WITH_NO_SCHEMA is addressed everywhere in the
+# code since, HANDLER_CLASS_NAMES_WHICH_STILL_NEED_SCHEMAS is temporary and
+# will be removed once every handlers in controller layer will become
+# ready for schema validation.
+HANDLER_CLASS_NAMES_WITH_NO_SCHEMA = (
+    HANDLER_CLASS_NAMES_WHICH_STILL_NEED_SCHEMAS +
+    HANDLER_CLASS_NAMES_WHICH_DO_NOT_REQUIRE_SCHEMAS)
```

Comment:
Changes due to merge conflict

Reply:
Yeah, this needs to be removed, `HANDLER_CLASS_NAMES_WHICH_STILL_NEED_SCHEMAS` is now moved to [handler_schema_constants.py](https://github.com/oppia/oppia/blob/develop/core/handler_schema_constants.py)